text/0000-lock-without-registry.md

```diff
@@ -0,0 +1,35 @@
+- Start Date: 2017-05-09
+- RFC PR:
+- Yarn Issue:
+
+# Summary
+
+The lockfile yarn.lock should not include the base registry (`https://registry.npmjs.org`).
+
+# Motivation
+
+In yarn.lock, the `resolved` field includes registry such as `https://registry.npmjs.org`.
+In China, most developers will set it to `https://registry.npm.taobao.org` for speed; but it seems slow for travis-ci and circleci.
+
+# Detailed design
+
+Replace the `resolved` by a `hash` field.
+The `url` in `resolved` is unnecessary; keeping `hash` is enough.
```

Comment:
Could we get an example of what the `resolved` field would look like following this standard? (Kind of like your examples in yarnpkg/yarn#3330)